packages/rulesets/src/spectral/az-arm.ts

```diff
@@ -538,6 +539,23 @@ const ruleset: any = {
       },
     },
 
+    ///
+    /// ARM RPC rules for operations API
+    ///
+
+    // RPC Code: RPC-Operations-V1-02
+    OperationsApiTenantLevelOnly: {
```

Comment:
I noticed that OperationsAPIImplementation ensures there is a path that matches regex `^/providers/[^/]+/operations$`. This rule I am adding would ensure there are no endpoints that end with `operations` besides the ones that match that regex. I did this to find operations endpoints that vary per subscription, because RPC-Operations-V1-02 says, 
> operations should *not* vary per subscription.

Though I can see how checking all paths that end with `operations` might cause problems, so maybe we need to reconsider if we want this rule. Also, maybe I should change it to ending with `/operations` instead of just `operations`.

I'm going to make this PR a draft for now.

Reply:
Updated it to look for `/operations`. I see some exceptions in the specs repo like

> `/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/apis/{apiId}/operations`

and

> `/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/databases/{databaseName}/operations`

So I might have to think about another way of checking for operations API endpoints that are not at tenant level.

Reply:
Changed it to check for /operations after a provider name at the subscription and resource group level. E.g.

> `/subscriptions/{subscriptionId}/providers/Microsoft.Bakery/operations`

and 

> `/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Bakery/operations`

Reply:
This should be sufficient for now